shared/desktop/app/installer.js

```diff
@@ -77,6 +79,8 @@ export default (callback: (err: any) => void): void => {
     }
 
     if (errorsResult.errors.length > 0) {
+      logger.info(errorsResult.errors.join('\n'))
+      logger.info(`Install errors: stdout=${stdout || ''}, stderr=${stderr || ''}`)
```

Comment:
I was concerned when I was working on this code that `stdout` and `stderr` wouldn't be defined in scope down here without an explicit `let`, but you'd know better than me of course, just mentioning it in case this is a mistake.

Reply:
its captured in the closure, lint would complain if thats not true